androidApp/src/main/res/values/strings.xml

```diff
@@ -98,5 +98,10 @@
     <string name="westbound">Westbound</string>
     <string name="stop_filter_pills_filter_hint">Apply a filter so that only arrivals from this route are displayed</string>
     <string name="stop_filter_pills_remove_hint">Remove the filter and display all routes</string>
-
+    <string name="refresh">Refresh</string>
+    <string name="refresh_predictions">Refresh predictions</string>
+    <string name="updated_mins_ago" tools:ignore="MissingTranslation">Updated %1$d minutes ago</string>
```

Comment:
question: Why is this missing translation? I think there should be an analogous string in use on iOS